fairseq/tasks/hubert_pretraining.py

```diff
@@ -134,18 +146,32 @@ def target_dictionary(self) -> Optional[Dictionary]:
     def dictionaries(self) -> List[Dictionary]:
         return [self._dictionaries[l] for l in self.cfg.labels]
 
+    @property
+    def pt_dictionaries(self) -> Optional[Dictionary]:
+        if self.fine_tuning:
+            logger.warning("Attempting to access pre-training dictionary during fine-tuning")
+        return self.state.pt_dictionaries
+
+    @property
+    def ft_dictioanries(self) -> Optional[Dictionary]:
```

Comment:
If you intend to return the dictionaries via the new properties `ft_dictionaries` and `pt_dictionaries` then you need to make sure that all callers of the property `dictionaries` are also changed.

However, I don't think you need two properties `ft_dictionaries` and `pt_dictionaries`. You can return the appropriate dictionary from the dictionaries property on line 146 based on `self.fine_tuning`.